### ISSUE TITLE
[3.9.x] [a11y] Search field has no label

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#24467]
### Steps to reproduce the issue
1. Use screen reader, e.g. NVDA
2. Go to the search results list page (for ex. use search module to search for a popular word, e.g. joomla on a test site)
3. Go to serach field
or
Inspect the code 
### Expected result
* The screen reader should announce a label for the search box

or
* The code should include a label for the field "Search" (label for="search-searchword"

### Actual result
Screen reader does not announce the name of the field
The field has no labels.

**Comment by hardik-codes** [comment]
Working on it

**Comment by hardik-codes** [comment]
@zwiastunsw 
``` id="q"``` is already there in 
```<input type="text" name="q" ```
then why the issue still persists?

**Comment by zwiastunsw** [comment]
I don't understand. You show a piece of code, but you don't write where it comes from.
This is piece of from "com_search/views/search/tmpl/default_form.php":
```
<div class="btn-group pull-left">
   <input type="text" name="searchword" title="<?php echo JText::_('COM_SEARCH_SEARCH_KEYWORD'); ?>" placeholder="<?php echo JText::_('COM_SEARCH_SEARCH_KEYWORD'); ?>" id="search-searchword" size="30" maxlength="<?php echo $upper_limit; ?>" value="<?php echo $this->escape($this->origkeyword); ?>" class="inputbox" />
</div>
```
In this code there is no label for="search-searchword" 
The label may be hidden for sighted users, but it must exist for users of screen readers. To hide the label from sighted users, use the `class="element-invisible"` for `label` tag.

**Comment by hardik-codes** [comment]
My bad 
I missed out giving the file name
Really sorry

**Comment by ghost** [comment]
closed as having Pull Request #24482
